http/servlet-undertow/src/test/java/io/quarkus/ts/servlets/HelloWorldQuarkusScenarioIT.java

```diff
@@ -0,0 +1,59 @@
+package io.quarkus.ts.servlets;
+
+import java.time.Duration;
+import java.util.Map;
+import org.apache.http.HttpStatus;
+import org.apache.http.util.Asserts;
+import org.junit.jupiter.api.Test;
+
+import io.quarkus.test.scenarios.QuarkusScenario;
+import io.restassured.RestAssured;
+
+import static io.quarkus.ts.listener.SessionListener.GAUGE_ACTIVE_SESSION;
+import static org.awaitility.Awaitility.await;
+
+@QuarkusScenario
+public class HelloWorldQuarkusScenarioIT {
+
+    static final Duration ACTIVE_SESSION_TIMEOUT = Duration.ofMinutes(2);
+    static final Duration REST_ASSURANCE_POLL_INTERVAL = Duration.ofSeconds(1);
+
+    @Test
+    public void undertowHelloWorld() {
```

Comment:
do we need this test?

the naming could change undertowHelloWorld => invokeUndertowServlet (better for naming in polarion)

Reply:
We don't need it, was leftover.